### PR TITLE
Updating CHANGELOG for release v0.3.0

### PR DESCRIPTION
**Pull request by kmadisa**
This PR serves to capture some of the major changes that were made to the library since the last release.

*Screenshots or code snippets (if appropriate):*
 - N/A

*Definition of Done Checklist*

- [x] Code meets our [python style guidelines](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit?usp=sharing)?
- [x] Unit tested (coded, passed, included)?
- [x] Requested at least 2 reviewers?
- [x] Commented code, particularly in hard-to-understand areas?
- [x] Made corresponding changes to the documentation (e.g. Python documentation, System Engineering Documentation, version description updates, README file, etc)?

JIRA: [N/A](https://skaafrica.atlassian.net/browse/JIRA_ISSUE_NUMBER)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 .. _fandango: https://github.com/tango-controls/fandango
 .. _79: https://github.com/ska-sa/tango-simlib/issues/79
 
+0.3.0
+-----
+- Support multiple device and control interfaces per server process (Enhancement)
+    - Addressing issue raised in #56
```
Comment (kmadisa) [inline CHANGELOG.rst]: Nice. Just some few things to do. Add a link to the issue like in line 2.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+- Adding a log when writing to the attributes
+- Reduced the unnecessary logging of the TANGO device
+- Handle non scalar value
+    - Addresses the  issue of not correctly reseting simulated quantity values to default
+    - When writing to a SPECTRUM type attribute, the data structure passed on to the write method is converted into an numpy          ndarray which was not handled. This was causing an error when it was trying to type cast an numpy.ndarray into an int.
```
Comment (kmadisa) [inline CHANGELOG.rst]: No need to add a long description here. Rather add a link to the issue/pull request. E.g. _Pull Request #209 (issue #207): Handle non scalar value_.

Comment (02Anele) [reply CHANGELOG.rst]: how do you add a link

Comment (kmadisa) [reply CHANGELOG.rst]: @02Anele Take a look at line 2 and line 25.

**Pull request merged**